samples/ProjectGrandmaster.Unity/Assets/Resources/Aydin's Chess set.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Is there a reason for naming folders after people? Should they be called Player1 or Player2?

Reply:
Those are the chess models created by that person. We'll get it renamed.

Reply:
Chess set deleted - Was not in use.